ws/mux_test.go

```
@@ -0,0 +1,244 @@
+}
+
+func TestServeHTTP_EchoWithAuthAndPathParams(t *testing.T) {
+	type captured struct {
+		user   string
+		id     string
+		params map[string]string
+	}
+	got := make(chan captured, 1)
+
+	m := newTestMux()
+	m.registerHandler("/ws/echo/{id}", route.GET, func(ctx context.Context, wsc *WSContext) {
```

golangci-lint (2.12.2)

[error] Error return value of `m.registerHandler` is not checked

(errcheck)